elliott

```diff
@@ -803,33 +842,78 @@ Example to look for bugs that have been moved back to MODIFIED
         # No bugs were provided
         raise click.BadParameter("If not using --auto then one or more --id's must be provided")
 
+    if len(id) == 0 and advisory is None and default_advisory_type is None:
+        # error, no bugs, advisory, or default selected
+        raise click.BadParameter("No input provided: Must use one of --id, --advisory, or --use-default-advisory")
+
     # Load bugzilla infomation and get a reference to the api
     runtime.initialize()
     bz_data = runtime.gitdata.load_data(key='bugzilla').data
     bzapi = elliottlib.bzutil.get_bzapi(bz_data)
-
     changed_bug_count = 0
     attached_bugs = []
 
+    # if --use-default-advisory is set we will override a user
```

Comment:
Can I recommend my approach in https://github.com/openshift/elliott/pull/37/commits/04b438767a92bed9faf788b7a05cb1edaec7aba1 rather than copy pasta...

Reply:
O u a smart guy

Reply:
I'll switch over to that after it's merginated. Don't wanna be causing no conflicts here.